### PR TITLE
[WIP] Updated the local installation file

### PR DESCRIPTION
**Pull request by codewithdev**
This PR brings the following changes in the https://actualbudget.org/docs/install/local page.

image

- Updated the content structure of the entire page.
- Removed unnecessary information and content
- Removed superficial information.

### CODE REVIEW

**Review by Teprifer — COMMENTED** [review]
Some good formatting changes here. :)

**Comment by codewithdev** [comment]
I will take a look at those review comments. Thanks for briefly reviewing this. Appreciate your efforts on this.

File: docs/install/local.md [hunk 1]
```diff
@@ -1,62 +1,51 @@
...
 ```bash
 yarn start
 ```
 
-Note that if you restart your computer, you’ll have to run this command again to start the server.
-
 ## Accessing Actual
 
-You should now be able to visit Actual by going to [http://localhost:5006](http://localhost:5006) in your browser.
+After the Actual server is started, you can access the Actual server from local server logged in the CLI as  [http://localhost:5006](http://localhost:5006).
```
[the remark this answers is not in the document]
Comment (RubenOlsen) [reply docs/install/local.md:43]: I agree with @Teprifer - but I also like some of the proposed changes. 

What about _After the servers have started, you can access Actual using your browser at [http://localhost:5006](http://localhost:5006). _

This is a bit more to the point compared to the original text and maybe a less confusing that the proposed change.

File: docs/install/local.md [hunk 2]
```diff
@@ -1,62 +1,51 @@
...
 ## Updating Actual
 
-When we publish a new release, you’ll need to follow these steps to update:
+Actual is constantly evolving to include new features and improve user's experience. It is recommended to update your local installation with our latest releases.
 
-1. Stop the server if it’s running. You can use the keyboard shortcut <Key mod="ctrl" k="c" /> (even on macOS) to stop the server, or close the terminal window it’s running from.
-2. Run `git pull` from the directory you cloned the project into. This will download the latest server code.
-3. Run `yarn install` from that same directory. This will download the latest web client code, along with any updated dependencies for the server.
-4. Restart the server by running `yarn start`.
+To update with our latest releases, use `git pull` command from the `master` branch of your local directory of Actual. After updating the changes, execute the `yarn install` to download the latest updates from the releases.
```
[the remark this answers is not in the document]
Comment (RubenOlsen) [reply docs/install/local.md:51]: I agree with @Teprifer - even if the proposed change is shorter, there are details that only seasoned people will understand, which is missing. We always need to remember that this part of the documentation is not necessarily for the "experts".

[the remark this answers is not in the document]
Comment (codewithdev) [reply docs/install/local.md:51]: Got it @RubenOlsen

**Comment by codewithdev** [comment]
Reviews have been incorporated.

File: docs/install/local.md [hunk 3]
```diff
@@ -37,15 +37,19 @@ After the Actual server is installed, start the Actual server by running the fol
 ```bash
 yarn start
 ```
+Note that if you restart your computer, you’ll have to run this command again to start the server.
 
 ## Accessing Actual
 
-After the Actual server is started, you can access the Actual server from local server logged in the CLI as  [http://localhost:5006](http://localhost:5006). 
+After servers have started, you can access Actual using your browser at [http://localhost:5006](http://localhost:5006).
```
Comment (Teprifer) [inline docs/install/local.md:44]: Slight tweak, should be 'server' singular, and has vs have, e.g.

After the server has been started

**Review by RubenOlsen — COMMENTED** [review]
Looking good to me.

**Review by RubenOlsen — APPROVED** [review]

File: docs/install/local.md [hunk 4]
```diff
@@ -1,62 +1,55 @@
...
 
-When we publish a new release, you’ll need to follow these steps to update:
-
-1. Stop the server if it’s running. You can use the keyboard shortcut <Key mod="ctrl" k="c" /> (even on macOS) to stop the server, or close the terminal window it’s running from.
+1. Stop the server if it’s running. You can use the keyboard shortcut <kbd>"ctrl" </kbd> <kbd>"c" </kbd> (even on macOS) to stop the server, or close the terminal window it’s running from.
 2. Run `git pull` from the directory you cloned the project into. This will download the latest server code.
 3. Run `yarn install` from that same directory. This will download the latest web client code, along with any updated dependencies for the server.
 4. Restart the server by running `yarn start`.
+
+Actual is constantly evolving to include new features and improve user's experience. It is always recommended to update your local installation with our [latest releases](https://actualbudget.org/docs/releases).
```
Comment (psybers) [inline docs/install/local.md:55]:
```suggestion
Actual is constantly evolving to include new features and improve the user's experience. It is always recommended that your local installation be updated with our [latest releases](https://actualbudget.org/docs/releases).
```

File: docs/install/local.md [hunk 5]
```diff
@@ -1,62 +1,55 @@
...
 
-When first accessing Actual, you may be prompted to provide a URL for the server. For a local installation like this, you can click the “Use localhost:5006” button to use the same URL as the one you’re accessing Actual from.
+When accessing Actual for the first time, you may be prompted to provide a URL for the server. For a local installation, click the **Use localhost:5006** button to use the server you've [configured](https://actualbudget.org/docs/config/).
 
 ## Updating Actual
 
-When we publish a new release, you’ll need to follow these steps to update:
-
-1. Stop the server if it’s running. You can use the keyboard shortcut <Key mod="ctrl" k="c" /> (even on macOS) to stop the server, or close the terminal window it’s running from.
+1. Stop the server if it’s running. You can use the keyboard shortcut <kbd>"ctrl" </kbd> <kbd>"c" </kbd> (even on macOS) to stop the server, or close the terminal window it’s running from.
```
Comment (psybers) [inline docs/install/local.md:50]:
```suggestion
1. Stop the server if it’s running. You can use the keyboard shortcut <kbd>CTRL-C</kbd> (even on macOS) to stop the server or close the terminal window it’s running from.
```

File: docs/install/local.md [hunk 6]
```diff
@@ -1,62 +1,55 @@
...
-```bash
-cd actual-server
-```
-
-Install all the dependencies using `yarn`
-
-```bash
-yarn install
-```
+2. Navigate to the Actual Server in your project root director.
```
Comment (psybers) [inline docs/install/local.md:25]:
```suggestion
2. Navigate to the Actual server in your project root director.
```

File: docs/install/local.md [hunk 7]
```diff
@@ -1,62 +1,55 @@
...
+    ```bash
+    cd actual-server
+    ```
+3. Install all the required dependencies using yarn.
```
Comment (psybers) [inline docs/install/local.md:29]:
```suggestion
3. Install all the required dependencies using `yarn`.
```

File: docs/install/local.md [hunk 8]
```diff
@@ -1,62 +1,55 @@
...
-npm install --global yarn
-```
+  ```bash
+  npm install --global yarn
+  ```
 
 ## Installing Actual
 
-First you need to clone the Actual Server project to your machine, you can do this using Git.
+1. After the prerequisites are fulfilled, clone the [Actual Server](https://github.com/actualbudget/actual-server) project in your project root directory where you want to install Actual.
```
Comment (psybers) [inline docs/install/local.md:20]:
```suggestion
1. After the prerequisites are fulfilled, clone the [Actual server](https://github.com/actualbudget/actual-server) project in your project root directory where you want to install Actual.
```

File: docs/install/local.md [hunk 9]
```diff
@@ -1,62 +1,55 @@
 # Local Installation
 
-The easiest way to get Actual running locally is to use the [actual-server](https://github.com/actualbudget/actual-server) project.
+The easiest way to get Actual running locally is to use [actual-server](https://github.com/actualbudget/actual-server). 
 
-Actual Server is the server element of Actual that is used for syncing changes across devices, and it comes with the latest version of Actual Web.
+Actual Server is the server element of Actual that is used for syncing changes across devices. It comes with the latest version of [Actual Web.](https://github.com/actualbudget/actual)
```
Comment (psybers) [inline docs/install/local.md:5]:
```suggestion
Actual server is used for syncing changes across devices. It comes with the latest version of the [Actual web client](https://github.com/actualbudget/actual).
```

File: docs/install/local.md [hunk 10]
```diff
@@ -1,62 +1,55 @@
...
 
-## Pre-requisites
+## Prerequisites
 
-Actual Server currently requires at least `Node.js` v18. If you don’t have Node.js installed, you can download the latest version of Node.js from the [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version). Consider using a tool like [`nvm`](https://github.com/nvm-sh/nvm) or [`asdf`](https://asdf-vm.com) to install and manage multiple versions of Node.js.
+- The Actual Server requires Node.js v18 or greater. You can download and install latest version of the Node.js from [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version).
```
Comment (psybers) [inline docs/install/local.md:9]:
```suggestion
- The Actual server requires Node.js v18 or greater. You can download and install the latest version of Node.js from [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version). 
```

**Review by psybers — CHANGES_REQUESTED** [review]

**Review by Teprifer — COMMENTED** [review]
@MatissJanis Got beaten to it by Psybers. Had a squiz through the suggested changes he has put forward and I'm happy with those so this is good to go once they've been incorporated.

**Review by RubenOlsen — APPROVED** [review]
Cleaning up abandoned PR with good content and incorporating proposed changes from @psybers

File: docs/install/local.md [hunk 11]
```diff
@@ -1,62 +1,51 @@
 # Local Installation
 
-The easiest way to get Actual running locally is to use the [actual-server](https://github.com/actualbudget/actual-server) project.
+The easiest way to get the Actual running on locally is to use the [actual-server](https://github.com/actualbudget/actual-server).
```
[the remark this answers is not in the document]
Comment (RubenOlsen) [reply docs/install/local.md]: Fixing this during cleanup of abandon PR

File: docs/install/local.md [hunk 12]
```diff
@@ -1,62 +1,51 @@
...
+The easiest way to get the Actual running on locally is to use the [actual-server](https://github.com/actualbudget/actual-server). 
 
-Actual Server is the server element of Actual that is used for syncing changes across devices, and it comes with the latest version of Actual Web.
+Actual Server is the server element of the Actual which is used for syncing changes across devices and which comes with the latest version of [Actual Web.](https://github.com/actualbudget/actual)
 
-## Pre-requisites
+## Prerequisites
 
-Actual Server currently requires at least `Node.js` v18. If you don’t have Node.js installed, you can download the latest version of Node.js from the [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version). Consider using a tool like [`nvm`](https://github.com/nvm-sh/nvm) or [`asdf`](https://asdf-vm.com) to install and manage multiple versions of Node.js.
+- The Actual Server requires Node.js v18 and greater. You can download and install latest version of the Node.js from [Node.js website](https://nodejs.org/en/download) (we recommend downloading the “LTS” version).
```
[the remark this answers is not in the document]
Comment (RubenOlsen) [reply docs/install/local.md]: Fixed during cleanup of abandoned PR.

**Pull request merged**